modules/gathering-data-specific-features.adoc

```diff
@@ -57,6 +57,12 @@ endif::openshift-dedicated[]
 |`registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel8:1.2.0`
 |Data collection for {sandboxed-containers-first}.
 
+|`registry.redhat.io/openshift4/poison-pill-must-gather-rhel8:v0.3.0`
```

Comment:
as discussed in chat, please use `v4.10`

Reply:
if we want to be consistent with the NMO and SNR must gather images, which don't have a `vX.Y` version but `vX.Y.Z` only, `v.4.10.1` might be a better choice, no clue... technically both are the same.